hydradx/tests/test_omnipool_amm.py

```diff
@@ -1490,3 +1496,187 @@ def test_dynamic_fees(hdx_price: float):
     if not final_usd_lrna_fee < intermediate_usd_lrna_fee:
         raise AssertionError('LRNA fee should decrease with time.')
 
+
+@given(
+    st.lists(asset_quantity_strategy, min_size=3, max_size=3),
+    st.lists(asset_quantity_bounded_strategy, min_size=3, max_size=3),
+    st.lists(asset_quantity_strategy, min_size=3, max_size=3),
+    st.lists(asset_quantity_strategy, min_size=3, max_size=3),
+    st.lists(asset_quantity_strategy, min_size=3, max_size=3),
+    st.lists(asset_price_strategy, min_size=2, max_size=2),
+    st.integers(min_value=10, max_value=1000),
+)
+def test_oracle_one_empty_block(liquidity: list[float], lrna: list[float], oracle_liquidity: list[float],
+                                oracle_volume_in: list[float], oracle_volume_out: list[float],
+                                oracle_prices: list[float], n):
+    alpha = 2 / (n + 1)
+
+    init_liquidity = {
+        'HDX': {'liquidity': liquidity[0], 'LRNA': lrna[0]},
+        'USD': {'liquidity': liquidity[1], 'LRNA': lrna[1]},
+        'DOT': {'liquidity': liquidity[2], 'LRNA': lrna[2]},
+    }
+
+    init_oracle = {
+        'liquidity': {'HDX': oracle_liquidity[0], 'USD': oracle_liquidity[1], 'DOT': oracle_liquidity[2]},
+        'volume_in': {'HDX': oracle_volume_in[0], 'USD': oracle_volume_in[1], 'DOT': oracle_volume_in[2]},
+        'volume_out': {'HDX': oracle_volume_out[0], 'USD': oracle_volume_out[1], 'DOT': oracle_volume_out[2]},
+        'price': {'HDX': oracle_prices[0], 'USD': 1, 'DOT': oracle_prices[1]},
+    }
+
+    state = oamm.OmnipoolState(
```

Comment:
I think calling this 'state' is confusing. It doesn't really matter because it isn't referenced again, but 'initial_omnipool' would be more consistent

Reply:
fixed